hub/powertoys/file-explorer.md

```diff
@@ -13,15 +13,18 @@ File Explorer add-ons currently include:
 
 - Preview Pane rendering of SVG icons (.svg)
 - Preview Pane rendering of Markdown files (.md)
+- Preview Pane rendering of source code (Monaco)
 - Preview Pane rendering of PDF files (.pdf)
+- Preview Pane rendering of G-code files (.gcode)
 - Icon thumb preview for SVG icons
 - Icon thumb preview for PDF files
+- Icon thumb preview for G-code files
```

Comment:
Let's have two levels here because it's easier to read:

- preview pain for:
   - svg
   - pdf
   - ...
- thumbnail for
   - g-code
   - ...

Reply:
Flip to (.cs, .cpp, .rust, ....) versus Monaco.  No one except a few will understand what Monaco is.

Reply:
sorry, .rs for rust